themes/jb/layouts/partials/head.html

```diff
@@ -5,18 +5,19 @@
   {{ partial "meta.html" . }}
 
   {{ $sass := resources.Get "css/main.sass" }}
+  {{ $video := resources.GetRemote "https://vjs.zencdn.net/7.18.1/video-js.min.css" }}
```

Comment:
I typically request people also `| minify | fingerprint` the files, so we can add some [security](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/a7585a36094491065cf88599b39c536eb68aad4e/themes/jb/layouts/partials/live/jb-tube.html#L6) (the `integrity` attribute) to the sources we're pulling in. Since this appears to already be minified (by the `.min` in the name), we shouldn't have to do the minification step.

```suggestion
  {{ $video := resources.GetRemote "https://vjs.zencdn.net/7.18.1/video-js.min.css" | fingerprint }}
```